pkg/reconciler/apis/apibinding/apibinding_controller.go

```diff
@@ -209,34 +209,33 @@ type controller struct {
 }
 
 // enqueueAPIBinding enqueues an APIBinding .
-func (c *controller) enqueueAPIBinding(obj interface{}) {
+func (c *controller) enqueueAPIBinding(obj interface{}, logSuffix string) {
 	key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
 	if err != nil {
 		runtime.HandleError(err)
 		return
 	}
 
-	klog.V(2).Infof("Queueing APIBinding %q", key)
+	klog.V(2).Infof("Queueing APIBinding %q%s", key, logSuffix)
 	c.queue.Add(key)
 }
 
 // enqueueAPIExport enqueues maps an APIExport to APIBindings for enqueuing.
-func (c *controller) enqueueAPIExport(obj interface{}) {
+func (c *controller) enqueueAPIExport(obj interface{}, logSuffix string) {
 	key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
 	if err != nil {
 		runtime.HandleError(err)
 		return
 	}
 
-	klog.V(2).Infof("Mapping APIExport %q", key)
 	bindingsForExport, err := c.apiBindingsIndexer.ByIndex(indexAPIBindingsByWorkspaceExport, key)
 	if err != nil {
 		runtime.HandleError(err)
 		return
 	}
 
-	for _, apiBinding := range bindingsForExport {
-		c.enqueueAPIBinding(apiBinding)
+	for _, obj := range bindingsForExport {
+		c.enqueueAPIBinding(obj, fmt.Sprintf(" because of APIExport %s%s", key, logSuffix))
```

Comment:
okay, you added an empty space, so it should be readable.

Reply:
yes, a pattern we should use everywhere IMO. X was enqueued because Y changed because Z changed.